internal/app/plan.go

```diff
@@ -141,25 +141,42 @@ func releaseWithHooks(cmd orderedCommand, storageBackend string, wg *sync.WaitGr
 		log.Verbose(err.Error())
 		return
 	}
+	var annotations []string
 	for _, c := range cmd.beforeCommands {
-		if err := execOne(c, cmd.targetRelease); err != nil {
+		if err := execOne(c.Command, cmd.targetRelease); err != nil {
 			errors <- err
+			if c.Type != "" {
```

Comment:
We shouldn't be validating this so late, per discussion -- will this ever be empty here? Should we be executing the command if it is empty?

Reply:
I've moved this to a function on the hook side, for now I think we should run it anyways but I think this can be cleaned up in a separate PR.